panoptes_aggregation/tests/batch_aggregation/test_batch_aggregation.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+from panoptes_aggregation.batch_aggregation import run_aggregation
+
+
+class TestBatchAggregation(unittest.TestCase):
+    def test_save_exports(self):
+        # Test that Panoptes calls are made and files are saved
+        assert 1 == 1
+
+    def test_process_wf_export(self):
+        # Test that:
+        # the wf export is parsed
+        # the version instance vars are set
+        # dataframe is retuned
+        assert 1 == 1
```

Comment:
A `patch` can be used to set a fixed input file stream and the test can check that it was parsed as expected.  This can either be defined in the test file (similar to the `extractor` and `reducer` tests), or an example `.csv` file can be placed in the test folder and read in (might be easier to debug and read in the long run).